bindings/python/dlite-collection-python.i

```diff
@@ -247,6 +247,10 @@ class Collection(Instance):
                 meta = metaid if isinstance(
                     metaid, dlite.Instance) else dlite.get_instance(
                         str(metaid).rstrip("#/"))
+                if not meta:
+                    raise TypeError(f"cannot instantiate metadata: {metaid}")
+                if meta.is_meta:
```

Comment:
I understand the errormessage, but why should meta.__class__ be set? Doesn't that happen when getting meta in 247-249?

Reply:
Good question. The DLite C level Python API represents all instances and metadata as a DLiteInstance. However, at the Python-level we have defined Metadata as a subclass of dlite.Instance with some additional functionality that only applies to metadata.  We want to have that extra functionality available in possible metadata returned by get_instances()